roles/openshift_hosted_logging/tasks/deploy_logging.yaml

```diff
@@ -165,6 +165,13 @@
   retries: 20
   delay: 15
 
+- name: Adding Kibana route information to loggingPublicURL
+  modify_yaml:
+    dest: "{{ openshift.common.config_base }}/master/master-config.yaml"
+    yaml_key: assetConfig.loggingPublicURL
+    yaml_value: "https://{{ openshift_logging_kibana_hostname }}"
+  notify: restart master
```

Comment:
We need to do this across all masters right? We probably need to push this up into a separate play in the playbook so that all masters get updated and restarted.